modules/nf-core/gsea/gsea/meta.yml

```diff
@@ -0,0 +1,138 @@
+name: "gsea_gsea"
+description: run the Broad Gene Set Enrichment tool in GSEA mode
+keywords:
+  - gene_set_analysis
+tools:
+  - "gsea":
+      description: "Gene Set Enrichment Analysis (GSEA)"
+      homepage: "http://www.gsea-msigdb.org/gsea/index.jsp"
+      documentation: "https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Main_Page"
+      doi: "10.1073/pnas.0506580102"
+      licence: "['BSD-3-clause']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing information on inputs and contrast, at a minimum
+        an ID, reference and target e.g. [ id:'test', reference:"WT",
+        target:"MUT" ]
+  - gct:
+      type: file
+      description: GCT file with expression values
+      pattern: "*.{gct}"
+  - cls:
+      type: file
+      description: CL file with the classes of the samples in the GCT file
+      pattern: "*.{gct}"
+  - gene_sets:
+      type: file
+      description: GMX or GMT file with gene sets
+      pattern: "*.{gmx,gmt}"
+  - ext.args2:
+      type: map
+      description: Map provided via ext in Nextflow configuration, defining
+        which of the groups defined in cls should be used for the reference and
+        target groups. Can be a closure referencing the meta.
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing information on inputs and contrast, at a minimum
+        an ID, reference and target e.g. [ id:'test', reference:"WT",
+        target:"MUT" ]
+  - report_tsvs:
```

Comment:
Please keep these entries in the same order as the outpu tchannels (tripped me up at first!, I thought you were missing abunch!)